nemo/collections/asr/models/aed_multitask_models.py

```diff
@@ -226,26 +238,24 @@ def transcribe(
         Returns:
             A list of transcriptions (or raw log probabilities if logprobs is True) in the same order as paths2audio_files
         """
-        if paths2audio_files is None or len(paths2audio_files) == 0:
+        if audio is None or len(audio) == 0:
             return {}
 
         if return_hypotheses:
             logging.warning("return_hypotheses=True is currently not supported, returning text instead.")
 
-        if isinstance(paths2audio_files, list):
-            logging.info(f"Found paths2audio_files to be a list of {len(paths2audio_files)} items.")
+        if isinstance(audio, list):
+            logging.info(f"Found paths2audio_files to be a list of {len(audio)} items.")
             logging.info(f"Assuming each item in paths2audio_files is a path to audio file.")
             logging.info(f"Transcribing with default Canary setting of English without PnC.")
```

Comment:
remove Canary reference ?

Reply:
Made it check if tokenizer is agg tokenizer, and then print first language name. Removed the stuff for PC - cant guarantee that tokenizer has support for PC or not.